action/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "test-observability-action",
+  "version": "0.1.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
```

Comment:
As far as I can tell we're not running this lint step in CI right now, probably should be added to `check.yml`